bots/machine/machine_core/ssh_connection.py

```diff
@@ -118,7 +118,7 @@ def wait_boot(self, timeout_sec=120):
             raise exceptions.Failure("Unable to reach machine {0} via ssh: {1}:{2}".format(self.label, self.ssh_address, self.ssh_port))
         self.boot_id = boot_id
 
-    def wait_reboot(self, timeout_sec=120):
+    def wait_reboot(self, timeout_sec=180):
```

Comment:
Bumping timouts is a red flag, but I think fixing the underlying issue here is hard (changing pcp's timeout?) and the commit message explains it well. Just saying :)